README.md

```diff
@@ -103,6 +103,7 @@ Add a vim-plug section to your `~/.vimrc` (or `~/.config/nvim/init.vim` for Neov
 ```vim
 " Specify a directory for plugins
 " - For Neovim: ~/.local/share/nvim/plugged
+" - For Neovim on Windows: ~/AppData/Local/nvim/autoload
```

Comment:
```md
- For Neovim: expand(stdpath('config') . '/nvim/plugged')
```

Reply:
`stdpath` is new to me but it looks nice. `stdpath('config')` on my system returns `/Users/jg/.config/nvim`, while `stdpath('data')` returns `/Users/jg/.local/share/nvim`. Should we use `stdpath(data)` without `expand` instead?

Reply:
`expand` is only for the backslash pathsep on Windows because of `noshellslash` but it should be redundant for vim-plug.

Reply:
>  Should we use `stdpath(data)` without `expand` instead?

Sure. `stdpath('data') . '/plugged'`.

Reply:
`stdpath('data') . '/plugged'` works. But `init.vim` is still loaded from `~/AppData/Local/nvim/init.vim`.